### ISSUE TITLE
Remove newline outputted with blank query

### ISSUE THREAD

**Issue by jinleileiking** [dbcli/mycli#248]
```
➜  ~ git:(master) ✗ mycli              
Version: 1.5.2
Chat: https://gitter.im/dbcli/mycli
Mail: https://groups.google.com/forum/#!forum/mycli-users
Home: http://mycli.net
Thanks to the contributor - Whitane Tech
mysql jinlei04@dbbk-baeond.xdb.all.serv:(none)> 








Time: 0.000s
mysql jinlei04@dbbk-baeond.xdb.all.serv:(none)> nopa
(1064, u"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'nopa' at line 1")
mysql jinlei04@dbbk-baeond.xdb.all.serv:(none)> nopager;
Pager disabled.
Time: 0.000s
mysql jinlei04@dbbk-baeond.xdb.all.serv:(none)> 

Time: 0.000s
mysql jinlei04@dbbk-baeond.xdb.all.serv:(none)> 

```

**Comment by jinleileiking** [comment]
version is 16-03-22.  github main version.

**Comment by jinleileiking** [comment]
Press enter will have a blank line before `Time: 0.000s`

**Comment by tsroten** [comment]
@jinleileiking Thanks for reporting this! Is the expected behavior this?
```
mysql jinlei04@dbbk-baeond.xdb.all.serv:(none)> 
Time: 0.000s
mysql jinlei04@dbbk-baeond.xdb.all.serv:(none)>
```

**Comment by jinleileiking** [comment]
@tsroten YES!

**Comment by tsroten** [comment]
@jinleileiking This is fixed as of [version 1.11.0](https://pypi.org/project/mycli/). If you installed via `pip`, you can upgrade by running `pip install --upgrade mycli`.

Thanks for the idea!
